railties/lib/rails/generators/rails/app/templates/config/initializers/backtrace_silencers.rb.tt

```diff
@@ -3,5 +3,6 @@
 # You can add backtrace silencers for libraries that you're using but don't wish to see in your backtraces.
 # Rails.backtrace_cleaner.add_silencer { |line| /my_noisy_library/.match?(line) }
 
-# You can also remove all the silencers if you're trying to debug a problem that might stem from framework code.
-# Rails.backtrace_cleaner.remove_silencers!
+# You can also remove all the silencers if you're trying to debug a problem that might stem from framework code
+# by setting BACKTRACE=1 before calling your invocation, like "BACKTRACE=1 ./bin/rails test".
```

Comment:
`rails test` isn’t the best example because it already has a `-b` option to print the full backtrace 😄

Reply:
Should have used the runner as an example. And preferably we’d just have one option that did it all.